mix/CodeModel.cpp

```diff
@@ -156,14 +161,19 @@ void CodeModel::runCompilationJob(int _jobId, QString const& _code)
 	emit compilationCompleteInternal(result.release());
 }
 
-void CodeModel::onCompilationComplete(CompilationResult*_newResult)
+void CodeModel::onCompilationComplete(CompilationResult* _newResult)
```

Comment:
Is CompilationResult* is instanciated using a QObject parent? if not maybe use a std:unique_ptr ...